tests/product-cart.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { test, expect } from '@playwright/test';
+
+test.beforeEach(async ({ page }) => {
+    await page.goto(process.env.WEB_URL as string);
+    await page.click('[data-test="product-01JR5X9HFW34R5EZ9D2SZ1H1GR"]');
```

Comment:
Seems this locator is not reliable, bacause 01JR5X9HFW34R5EZ9D2SZ1H1GR can be different for environments